utils/entity.js

```diff
@@ -106,6 +108,7 @@ function prepareEntityForTemplates(entityWithConfig, generator) {
   entityWithConfig.resetFakerSeed = (suffix = '') =>
     entityWithConfig.faker.seed(stringHashCode(entityWithConfig.name.toLowerCase() + suffix));
   entityWithConfig.resetFakerSeed();
+  entityWithConfig.getPrimaryKeyValue = pk => generator.getPrimaryKeyValue(pk);
```

Comment:
Is there a reason for adding this?
It will override the generator one, removing others arguments.
```suggestion
```

Reply:
There was one, because I didn't understand the way it worked... Currently on it, expect a commit in 5 minute (the time unit test will execute and succeed).
Sorry for the inconvenience

Reply:
No problem.

Reply:
The good thing is that this templates review will help to remove dependencies to context (at least on the template side).
So we'll have only one source for a particular need in the end